doseCalc/+DoseEngines/matRad_MonteCarloEngineAbstract.m

```diff
@@ -90,14 +90,15 @@ function setDefaults(this)
             this.calcDoseDirect = true;
             dij = this.calcDose(ct,cst,stf);
 
-            % hack dij struct
-            dij.numOfBeams = 1;
-            dij.beamNum = 1;
-
-            % calculate cubes; use uniform weights here, weighting with actual fluence
-            % already performed in dij construction
-            resultGUI    = matRad_calcCubes(sum(w),dij);
-
+            if ~this.externalCalculation
```

Comment:
This fails the test because externalCalculation is not a property of the base class, and thus MCsquare calculations will not run as the property is unknown. I don't know if it makes sense to force the existance of this.externalCalculation in all MC engines. I would rather extend/overwrite the calcDoseForward function in the derived TOPAS engine such that the external calculation switch is handled there.